selfhost/typechecker.jakt

```diff
@@ -78,13 +78,13 @@ struct TypeId {
     function from_string(anon type_id_string: String) throws -> TypeId {
         let parts = type_id_string.split('_')
         if not (parts.size() == 2) {
-            throw Error::from_errno(999) // FIXME: good error message
+            panic(format("Failed to convert string `{}` to a TypeId: Wrong number of parts. (Wanted 2, got {})", type_id_string, parts.size()))
```

Comment:
I can't comment on the commit message directly. The header says `Fix inverted condition`, but you only change from error to panic with a message, I see no change in conditions.

Reply:
Good catch! It *did* fix the condition, but then someone else did and got their fix merged first. I am being too slow. :sweat_smile: